public/index.html

```diff
@@ -1,17 +1,42 @@
-<title>devPunks \m/</title> 
+<title>devPunks \m/</title>
 
 <link rel='stylesheet' href='/style.css'>
 
 <h1>devPunks \m/</h1>
-<h2>The first technology show from junior to super senior developer in 15 minutes.</h2>
 
-<blockqoute>
-</blockquote>
+<h2>Leveling the playing field, developer to developer.</h2>
+
 
 <main>
   <img src='/quote.png' alt='Dev Punks'>
 
+  <section id='mission'>
+
+  <p>The devPunk subculture, includes a diverse array of ideologies, fashions and forms of expression, including visual art, literature, and film. The subculture is largely characterized by <strong><u>anti-establishment</u> views and the promotion of <u>individual freedom</u></strong>. The devPunk subculture is centered on a loud, aggressive genre of developers. It is usually played by small teams consisting of a junior, one or two seniors, and a hiring manager.
```

Comment:
@melaniegershman what about "A developer and their mentor" removes this junior/senior bullshit. Death to it truth be told!